src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,81 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+import static seedu.addressbook.common.Messages.*;
+
+public class Formatter {
```

Comment:
This class and its non-trivial methods should have header comments.
If not, developers who write client code for this class will not know how to use it.